views/update/update-layout.html

```diff
@@ -98,7 +98,7 @@
 {% endblock %}
 
 {% block bodyEnd %}
-  <script src="//{{CDN_HOST}}/javascripts/govuk-frontend/v3.11.0/govuk-frontend-3.11.0.min.js"></script>
+  <script src="//{{CDN_HOST}}/javascripts/govuk-frontend/v4.6.0/govuk-frontend-4.6.0.min.js"></script>
```

Comment:
I think we should probably do this update in a separate PR. I think we have a ticket for it. There are a couple of other places to replace with 4.6: the css on this page and the js and css on `layout.html`

Toro Loco also want us to update the package in `packages.json` to 4.6 then run nmp -i and commit the changes. I think this deserves it's own PR... also makes it easier from a testing perspective